debian/control

```diff
@@ -4,12 +4,12 @@ Priority: important
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: bash-completion,
                debhelper (>=9),
-               debianutils,
+               debianutils (>= 4.7),
                dh-python,
-               dh-systemd | debhelper (>= 13.3),
+               debhelper (>= 13.3) | dh-systemd,
```

Comment:
Whilst you define `debhelper (>= 13.3)` here, be aware that there exists d/compat with value 9 there.
Because of this, if you inspect the build logs, you'll see the following warnings everywhere:
`dh_auto_build: warning: Compatibility levels before 10 are deprecated (level 9 in use)`

This essentially means that even in impish, the compat level 9 is being used and might defeat the whole purpose for having a strict requirement of "debhlper (>= 13.3)".

Reply:
Hi Utkarsh, I think we need to live with that warning message if we want to keep a single d/compat that works across all the supported releases.

Reply:
Sure, I just wanted to mention that so that it's known, at least.
OTOH, I was curious if we have a d/compat with 9 mentioned and debhlper (>= 13.3), which of the following is gonna be used, really? 😮

Reply:
I think it depends on which target release is being build, since dh-systemd is unavailable on (groovy and later) I think the deb-helper >= 13.3 is used. This compexity in d/control is probably something that future-us needs to fragment a bit so that each d/control file is simple and better representative of the desired depends on each Ubuntu release. We can touch on this later though I think.

Reply:
Note that given I can't remember which series drops the dh-systemd deps it adds more credence to our needig making this easier (or better documented) in the future.